lib/matplotlib/artist.py

```diff
@@ -1593,52 +1597,59 @@ def getp(obj, property=None):
+        only the first instance in the sequence is queried.
 
-      >>> line, = plot([1, 2, 3])
-      >>> setp(line, linestyle='--')
+        For example, two lines can be made thicker and red with a single call:
 
-    If you want to know the valid types of arguments, you can provide
-    the name of the property you want to set without a value::
+        >>> x = arange(0, 1, 0.01)
+        >>> lines = plot(x, sin(2*pi*x), x, sin(4*pi*x))
+        >>> setp(lines, linewidth=2, color='r')
 
-      >>> setp(line, 'linestyle')
-          linestyle: {'-', '--', '-.', ':', '', (offset, on-off-seq), ...}
+    file : file-like or None, default: `sys.stdout`
+        Where `setp` writes its output, e.g. ::
+
+            with open('output.log') as file:
+                setp(line, file=file)
 
-    If you want to see all the properties that can be set, and their
-    possible values, you can do::
+    *args, **kwargs
+        The properties to set.  The following combinations are supported:
 
-      >>> setp(line)
-          ... long output listing omitted
+        - Set the linestyle of a line to be dashed:
 
-    By default `setp` prints to `sys.stdout`, but this can be modified using
-    the *file* keyword-only argument::
+          >>> line, = plot([1, 2, 3])
+          >>> setp(line, linestyle='--')
+
+        - Set multiple properties at once:
+
+          >>> setp(line, linewidth=2, color='r')
+
+        - List allowed values for a line's linestyle:
+
+          >>> setp(line, 'linestyle')
+          linestyle: {'-', '--', '-.', ':', '', (offset, on-off-seq), ...}
 
-      >>> with fopen('output.log') as f:
-      >>>     setp(line, file=f)
+        - List all properties that can be set, and their allowed values:
 
-    :func:`setp` operates on a single instance or a iterable of
-    instances. If you are in query mode introspecting the possible
-    values, only the first instance in the sequence is used. When
-    actually setting values, all the instances will be set.  e.g.,
-    suppose you have a list of two lines, the following will make both
-    lines thicker and red::
+          >>> setp(line)
+          agg_filter: a filter function, ...
+          [long output listing omitted]
 
-      >>> x = arange(0, 1, 0.01)
-      >>> y1 = sin(2*pi*x)
-      >>> y2 = sin(4*pi*x)
-      >>> lines = plot(x, y1, x, y2)
-      >>> setp(lines, linewidth=2, color='r')
+        `setp` also supports MATLAB style string/value pairs.  For example, the
+        following are equivalent:
 
-    :func:`setp` works with the MATLAB style string/value pairs or
-    with python kwargs.  For example, the following are equivalent::
+        >>> setp(lines, 'linewidth', 2, 'color', 'r')  # MATLAB style
+        >>> setp(lines, linewidth=2, color='r')        # Python style
 
-      >>> setp(lines, 'linewidth', 2, 'color', 'r')  # MATLAB style
-      >>> setp(lines, linewidth=2, color='r')        # python style
+    See Also
+    --------
+    getp
```

Comment:
```suggestion
    `.getp`
```

Reply:
This is already linked, because it's in the same file, I think.

Reply:
You're right, should have waited for it to build locally to check.